route/route.md

```diff
@@ -34,6 +26,16 @@ should start with either a character (a-Z) or colon (:).  Examples:
 
 @body
 
+## Video
```

Comment:
Can you change this to `## Use`.  This way, we are consistent with every other page.  Thanks!